File: ProjectEuler.net/Python/06-LargestPalindromeProduct.py
'''
Find the largest palindrome made from the product of two 3-digit numbers.
'''
c = 999 * 999
while c > (100 * 1000) :
    if (c % 10) == ((int)(c / 100000)) :
        if ((int)((c % 100) / 10)) == ((int)(c / 10000)) % 10 :
            if ((int)((c % 1000) / 100)) == ((int)(c / 1000)) % 10 :
                for i in range (999, 99, -1) :
                    if (c % i) == 0 :
                        if ((int)(c / i)) < 1000 and ((int)(c / i)) > 100 :
                            print (c)
                            exit ()
    c -= 1

File: ProjectEuler.net/Swift/06-LargestPalindromeProduct.swift
/*
Find the largest palindrome made from the product of two 3-digit numbers.
*/
var a = Int (999 * 999)
var b = Int (1)
while (a > 100 * 1000) {
    if (a % 10) == (Int(a / 100000)) {
        if (Int((a % 100) / 10)) == ((Int(a / 10000)) % 10) {
            if (Int((a % 1000) / 100)) == ((Int(a / 1000)) % 10) {
                for i in stride(from: 999, through: 100, by: -1) {
                    if (a % i) == 0 {
                        if ((Int(a / i)) < 1000) && ((Int(a / i)) > 100) {
                            print (a)
                            b = 0
                            break
                        }
                    }
                }
                if (b == 0) {
                    break
                }
            }
        }
    }
    a -= 1
}